packages/server/src/tests/banners/bannerTargetingTests.ts

```diff
@@ -1,4 +1,34 @@
 import { TargetingTest } from '../../lib/targetingTesting';
 import { BannerTargeting } from '@sdc/shared/types';
 
-export const bannerTargetingTests: TargetingTest<BannerTargeting>[] = [];
+const isNetworkFront = (targeting: BannerTargeting): boolean =>
+    // https://github.com/guardian/frontend/blob/5b970cd7308175cfc1bcae2d4fb8c06ee13c5fa0/common/app/model/DotcomContentType.scala#L33
+    targeting.contentType?.toLowerCase() === 'network front';
+
+export const bannerTargetingTests: TargetingTest<BannerTargeting>[] = [
+    {
+        name: '2022-02-10_BannerTargeting_PageView',
+        // Exclude browsers that have not consented to article counting
+        canInclude: (targeting: BannerTargeting) => targeting.articleCountToday !== undefined,
```

Comment:
this condition also means no browsers will enter the test until DCR/frontend have been updated to send the new `articleCountToday` field